Added Logging Configuration (#218)
microsoft/ApplicationInsights-Python: azure-monitor-opentelemetry-distro/README.md

### Key Concepts

> This package bundles a collection of OpenTelemetry and Azure Monitor components to enable the collection and sending of telemetry to Azure Monitor. For MANUAL instrumentation, use the `configure_opentelemetry` function. AUTOMATIC instrumentation is not yet supported.

Reviewer: ```suggestion
This package bundles a series of OpenTelemetry and Azure Monitor components to enable the collection and sending of telemetry to Azure Monitor. For MANUAL instrumentation, use the `configure_azure_monitor` function. AUTOMATIC instrumentation is not yet supported.
```

pip install azure-monitor-opentelemetry-distro --pre
```

> ### Manual Configuration

Reviewer: ```suggestion
### Usage
```

You can use the `configure_opentelemetry` to set up manual instrumentation in a single line and send telemetry for your app to Azure Monitor. `configure_opentelemetry` supports the following optional arguments:

> 

Reviewer: No need to mention specifically about exporter implementation detail. Just say: `connection string will be automatically populated from the APPLICATIONINSIGHTS_CONNECTION_STRING environment variable if not explicitly passed in.`

Author: Done

> * connection_string - You can pass in your connections string here. If left out, the [Azure Monitor OpenTelemetry exporters][azure_monitor_opentelemetry_exporters] will use the connection string specified in the `APPLICATIONINSIGHTS_CONNECTION_STRING` environment variable. 

Reviewer: ```suggestion
* connection_string - You can pass in your connection string here. If left out, the [Azure Monitor OpenTelemetry exporters][azure_monitor_opentelemetry_exporters] will use the connection string specified in the `APPLICATIONINSIGHTS_CONNECTION_STRING` environment variable. 
```

> * connection_string - You can pass in your connections string here. If left out, the [Azure Monitor OpenTelemetry exporters][azure_monitor_opentelemetry_exporters] will use the connection string specified in the `APPLICATIONINSIGHTS_CONNECTION_STRING` environment variable. 

Reviewer: As part of these definitions, it would be great if we could link to some external documentation referencing to what these things are. For example:

[logging_level][logging_level_doc]

[logging_level_doc]: https://docs.python.org/3/library/logging.html#levels

The service related ones can be found [here](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#service)

Author: Done

* service_name = Specefies the service name. 
* service_namespace = Specefies the service namespace.
* service_instance_id = Specefies the instance id.

> * disable_tracing = If set to `True`, disables collection and export or trace telemetry

Reviewer: ```suggestion
* disable_tracing = If set to `True`, disables collection and export of distributed tracing telemetry
```

> * disable_tracing = If set to `True`, disables collection and export or trace telemetry

Reviewer: The exporter related configurations are missing from here but you can add a TODO and create a tracking issue for this.

Author: Added logging_level, sampling_ratio, and export intervals

Reviewer: I meant linking to [this](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/monitor/azure-monitor-opentelemetry-exporter#configuration). Have a section at the bottom saying something like: "Additional configuration related to exporting: "

Author: Done

* logging_level = Determines the logging level of the Opentelemetry Logging Handler. Ex: logging.WARNING

> * disable_logging = If set to `True`, disables collection and export or trace telemetry

Reviewer: ```suggestion
* disable_logging = If set to `True`, disables collection and export of logging telemetry
```

### Prerequisites:
To use this package, you must have:

> * Azure subscription - [Create a free account][azure_sub]

Reviewer: Nit: `application_insights_namespace` link is broken. Replace with [this](https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview)

Author: Done

> * Azure subscription - [Create a free account][azure_sub]

Reviewer: Add an entry for `connection_string_doc` and use [this](https://learn.microsoft.com/en-us/azure/azure-monitor/app/sdk-connection-string) link.

Author: Done

pip install azure-monitor-opentelemetry-distro --pre
```

### Usage

You can use `configure_azure_monitor` to set up instrumentation for your app to Azure Monitor. `configure_azure_monitor` supports the following optional arguments:

* connection_string - The [connection string][connection_string_doc] for your Application Insights resource. The connection string will be automatically populated from the `APPLICATIONINSIGHTS_CONNECTION_STRING` environment variable if not explicitly passed in.

> * service_name = Specifies the service name. 

Reviewer: Link to this [doc](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#service) for service_*

Author: done

* service_namespace = Specifies the service namespace.
* service_instance_id = Specifies the instance id.
* disable_logging = If set to `True`, disables collection and export of logging telemetry.
* logging_level = Specifies the [logging level][logging_level] of the Opentelemetry Logging Handler. Ex: logging.WARNING.
* logging_export_interval_millis = Specifies the export interval of the logging exporter in milliseconds. Defaults to 30,000.
* disable_tracing = If set to `True`, disables collection and export of distributed tracing telemetry.

> * sampling_ratio = Specifies the ratio of distributed tracing telemetry to be sampled. Defaults to 1.0, meaning no distributed tracing telemetry is filtered out.

Reviewer: ```suggestion
* sampling_ratio = Specifies the ratio of distributed tracing telemetry to be [sampled][application_insights_sampling]. Accepted values are in the range [0,1]. Defaults to 1.0, meaning no telemetry is sampled out.
```

Author: Done manually.

[azure_monitor_opentelemetry_exporters]: https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/monitor/azure-monitor-opentelemetry-exporter#microsoft-opentelemetry-exporter-for-azure-monitor
[azure_portal]: https://portal.azure.com
[azure_sub]: https://azure.microsoft.com/free/
[application_insights_namespace]: https://learn.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview

> [connection_string_doc]: https://learn.microsoft.com/en-us/azure/azure-monitor/app/sdk-connection-string

Reviewer: [application_insights_sampling]: https://learn.microsoft.com/en-us/azure/azure-monitor/app/sampling
